data/mainnet/tokens.js

```diff
@@ -5546,4 +5546,106 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['iFARM', 'DAI', 'WETH'],
   },
+  bal_TUSD_STABLE: {
+    category: VAULT_CATEGORIES_IDS.BALANCER,
+    chain: CHAINS_ID.MATIC_MAINNET,
+    isNew: true,
+    logoUrl: './icons/bal-tusd.png',
+    apyIconUrls: [],
+    apyTokenSymbols: ['BAL'],
+    displayName: 'Balancer: TUSD STABLE',
+    subLabel: 'TUSD',
+    tokenAddress: addresses.MATIC.V2.bal_TUSD_STABLE.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.MATIC.V2.bal_TUSD_STABLE.NewVault,
+    priceFunction: {
+      type: GET_PRICE_TYPES.BALANCER,
+      params: [
+        addresses.MATIC.V2.bal_TUSD_STABLE.Underlying,
+        addresses.MATIC.V2.bal_TUSD_STABLE.PoolId,
+        CHAINS_ID.MATIC_MAINNET,
+      ],
+    },
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.BALANCER,
+        params: [
+          addresses.MATIC.V2.bal_TUSD_STABLE.Underlying,
+          addresses.MATIC.V2.bal_TUSD_STABLE.PoolId,
+          '1000',
```

Comment:
This number is wrong, please check out this public sheet where the reward amounts for the balancer pools are announced weekly: [https://docs.google.com/spreadsheets/d/1XDwShNDhKKzINtwBja7TsOw34JIwYktjGPLiG0WQl1s/edit?usp=sharing](https://docs.google.com/spreadsheets/d/1XDwShNDhKKzINtwBja7TsOw34JIwYktjGPLiG0WQl1s/edit?usp=sharing)